addons/fortify/XEH_preStart.sqf

```diff
@@ -1,3 +1,9 @@
 #include "script_component.hpp"
 
 #include "XEH_PREP.hpp"
+
+private _fortifyTools = (QUOTE(getNumber (_x >> QQGVAR(fortifyTool)) > 0) configClasses (configFile >> "CfgWeapons") apply {configName _x});
+_fortifyTools append    (QUOTE(getNumber (_x >> QQGVAR(fortifyTool)) > 0) configClasses (configFile >> "CfgVehicles") apply {configName _x});
+TRACE_1("",_fortifyTools);
+
+uiNamespace setVariable [QGVAR(fortifyTools), compileFinal str _fortifyTools];
```

Comment:
```suggestion
uiNamespace setVariable [QGVAR(fortifyTools), compileFinal (_fortifyTools createHashMapFromArray [])];
```

Reply:
What's the point in making it a hashmap with no values, for (potential) future functionality?

Reply:
Oh probably to just not make it modifiable in preStart?

Reply:
That and it's faster to look up.

Reply:
Yeah I realized I misunderstood the caching at first, I thought it was saving the result of the lookup, rather than the function itself